dpnp/dpnp_iface_arraycreation.py

```diff
@@ -1206,7 +1278,7 @@ def empty_like(
     /,
     *,
     dtype=None,
-    order="C",
+    order="K",
```

Comment:
As I can see, in numpy, the default value is `None`.
And we need to explain what none value means, that info is currently missing.